docs/command_list/newt_mfg.rst

```diff
@@ -87,16 +87,25 @@ The ``rb_blinky_rsa`` package needs a pkg.yml file. In addition it is needs a mf
 .. code-block:: console
 
     $  more mfgs/rb_blinky_rsa/mfg.yml
-    mfg.bootloader: 'targets/rb_boot'
-    mfg.images:
-        - 'targets/rb_blinky'
+    mfg.bsp: "@apache-mynewt-core/hw/bsp/rb-nano2"
+    mfg.targets:
+        - rb_boot:
+          name: "targets/rb_boot"
+          area: FLASH_AREA_BOOTLOADER
+          offset: 0x0
+        - rb_blinky:
+          name: "targets/rb_blinky"
+          area: FLASH_AREA_IMAGE_0
+          offset: 0x0
+    mfg.meta:
+        area: FLASH_AREA_BOOTLOADER
 
 Build the bootloader and app images.
 
 .. code-block:: console
 
     $ newt build rb_boot
-    $ newt create-image rb_blinky 0.0.1
+    $ newt build rb_blink && newt create-image rb_blinky 0.0.1
```

Comment:
That should be `rb_boot` (not `rb_blink`), right? :)

Reply:
Thanks, @ccollins476ad. You are right ;) I am a newbie to newt... Changed.